processing/pom.xml

```diff
@@ -44,6 +44,10 @@
             <artifactId>emitter</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.metamx</groupId>
```

Comment:
this is introducing sigar back as a dependency, which we explicitly removed because not everyone wants it